sdk/openai/Azure.AI.OpenAI/src/Custom/AzureOpenAIAudience.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.ComponentModel;
+
+namespace Azure.AI.OpenAI;
+
+/// <summary>
+/// Represents cloud authentication audiences available for Azure OpenAI.
+/// These audiences correspond to authorization token authentication scopes.
+/// </summary>
+public readonly partial struct AzureOpenAIAudience : IEquatable<AzureOpenAIAudience>
+{
+    private readonly string _value;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="AzureOpenAIAudience"/> object.
+    /// </summary>
+    /// <remarks>
+    /// Please consider using one of the known, valid values like <see cref="AzurePublic"/> or <see cref="AzureGovernment"/>.
+    /// </remarks>
+    /// <param name="value">
+    /// The Microsoft Entra audience to use when forming authorization scopes.
+    /// For Azure OpenAI, this value corresponds to a URL that identifies the Azure cloud where the resource is located.
+    /// For more information: <see href="https://learn.microsoft.com/azure/azure-government/documentation-government-cognitiveservices" />.
+    /// </param>
+    /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+    public AzureOpenAIAudience(string value)
+    {
+        Argument.AssertNotNullOrEmpty(value, nameof(value));
+        _value = value;
+    }
+
+    private const string AzurePublicCloudValue = "https://cognitiveservices.azure.com/.default";
+    private const string AzureGovernmentCloudValue = "https://cognitiveservices.azure.us/.default";
+
+    /// <summary>
+    /// The authorization audience used to connect to the public Azure cloud. Default if not otherwise specified.
+    /// </summary>
+    public static AzureOpenAIAudience AzurePublic { get; } = new AzureOpenAIAudience(AzurePublicCloudValue);
```

Comment:
This one is actually called "AzurePublicCloud". I think it's a marketing inconsistency. :/

Reply:
I... wha... OK, OK, renamed!